test/askContainer/treeViews/treeViewProviders/helpViewProvider.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
+import * as assert from "assert";
+import * as sinon from "sinon";
+
+import { HelpViewProvider } from "../../../../src/askContainer/treeViews/treeViewProviders/helpViewProvider";
+import { HelpView } from "../../../../src/askContainer/treeViews/helpView";
+import { ext } from "../../../../src/extensionGlobals";
+import { PluginTreeItem, CustomResource, ContextValueTypes, Resource } from '../../../../src/runtime/';
+import { HELP_VIEW_ITEMS, EXTERNAL_LINKS } from '../../../../src/constants';
+import { Logger } from '../../../../src/logger';
+
+describe("TreeView_helpViewProvider tests", () => {
+    let helpView: HelpView;
+    let helpViewProvider: HelpViewProvider;
+    let sandbox: sinon.SinonSandbox;
+    let debugSpy: sinon.SinonStub;
+    let verboseSpy: sinon.SinonStub;
+    before(() => {
+        helpView = new HelpView(ext.context);
+        helpViewProvider = new HelpViewProvider(helpView);
+    });
+
+    beforeEach(() => {
+        sandbox = sinon.createSandbox();
+        debugSpy = sandbox.stub(Logger, 'debug');
+        verboseSpy = sandbox.stub(Logger, 'verbose');
```

Comment:
I don't think it makes a lot of sense spying the logger actions. We can definitely check logger by itself, by adding unit tests. Stubbing is fine, in case we don't want the actual debug outputs to be added. But asserting on logger values are an overkill and can be safely removed IMO. Let me know if you think otherwise.